packages/wrangler/src/config/validation-pages.ts

```diff
@@ -38,6 +38,7 @@ const supportedPagesConfigFields = [
 	"browser",
 	// normalizeAndValidateConfig() sets this value
 	"configPath",
+	"upload_source_maps",
```

Comment:
Does this feature require `wrangler.toml`? If so, it's the first Pages feature that doesn't work without a config file—is that okay?

Reply:
I've added a `--upload-source-maps` flag for the deploy command and used the existing `--sourcemap` flag for the internal pages function build command. The explicit argument will take precedence over whatever is in the wrangler.toml